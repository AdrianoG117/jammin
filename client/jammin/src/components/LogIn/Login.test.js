import {render, screen} from '@testing-library/react';
import Login from './LogIn';


describe('Signup component', ()=> {
  
  test('should match the snapshot', () => {
    const {container} = render(<Login
                  setUserData={()=>{}}
                  setIsSignedUp={()=>{}} />);
    expect(container.firstChild).toMatchSnapshot();
  })
  
  test('Should render the headings', () => {
    render (<Login
                  setUserData={()=>{}}
                  setIsSignedUp={()=>{}} />);
    //title
    screen.getByPlaceholderText(/Email/);
    screen.getByPlaceholderText(/Password/);
    screen.getByRole('button', { name: 'LOG IN' });
  })

})

// add tests for both functions

