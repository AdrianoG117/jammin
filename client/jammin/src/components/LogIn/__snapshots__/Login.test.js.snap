// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component should match the snapshot 1`] = `
<div
  class="login-container"
>
  <form
    class="login-form"
  >
    <input
      id="mail"
      name="email"
      placeholder="Email"
      required=""
      type="text"
      value=""
    />
    <input
      id="pwd"
      name="password"
      placeholder="Password"
      required=""
      type="password"
      value=""
    />
    <button
      class="input-btn"
    >
      LOG IN
    </button>
  </form>
</div>
`;
